internal/core/metadata/rest_deviceprofile.go

```diff
@@ -533,14 +533,6 @@ func restGetYamlProfileByName(
 	w.Write(out)
 }
 
-/*
- * Implementation: https://groups.google.com/forum/#!topic/golang-nuts/EZHtFOXA8UE
- * Response:
- * 	- 200: database generated identifier for the new device profile
- *	- 400: YAML file is empty
- *	- 409: an associated command's name is a duplicate for the profile or if the name is determined to not be uniqe with regard to others
- * 	- 503: Server Error
- */
 func restGetYamlProfileById(
```

Comment:
Removed a comment; expectation was that you'd replace it with a docblock comment that says what restGetYamlProfileById does.